buildSrc/src/main/java/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -323,69 +295,52 @@ private static Configuration configureExamplePlugin(Project project) {
         return examplePlugin;
     }
 
-    private static TaskProvider<GradleDistroTestTask> configureVMWrapperTask(
+    private static void configureWrapperTasks(
```

Comment:
Why rename this? I think having "VM" in the name is helpful. Else let's add a comment.